contracts/sidechain/L2PoolManagerProxy.sol

```diff
@@ -3,24 +3,35 @@ pragma solidity 0.8.11;
 
 import { NonblockingLzApp } from "../layerzero/lzApp/NonblockingLzApp.sol";
 import { IPoolManagerLite } from "contracts/sidechain/interfaces/IPoolManagerLite.sol";
+import { KeeperRole } from "../peripheral/KeeperRole.sol";
 
 /**
  * @title   L2PoolManagerProxy
  * @author  AuraFinance
  * @dev     Given a root gauge on L1PoolManagerProxy it adds a gauge recipient on PoolManagerLite
  */
-contract L2PoolManagerProxy is NonblockingLzApp {
+contract L2PoolManagerProxy is NonblockingLzApp, KeeperRole {
```

Comment:
The KeeperRole  passes on the constructor the msg.sender what will make this contract to have different address on each chain.

Reply:
should deploy with the same address no? Assuming msg.sender is the same which it will be because its the factory:

`address = keccak256(0xff ++ sender_address ++ salt ++ keccak256(init_code))[12:]`